ng2-components/ng2-alfresco-documentlist/README.md

```diff
@@ -192,6 +192,7 @@ The properties currentFolderId, folderNode and node are the entry initialization
 | `rowFilter` | `RowFilter` | | Custom row filter, [see more](#custom-row-filter). |
 | `imageResolver` | `ImageResolver` | | Custom image resolver, [see more](#custom-image-resolver). |
 | `allowDropFiles` | boolean | false | Toggle file drop support for rows (see **ng2-alfresco-core/UploadDirective** for more details) |
+| `sorting` | string[] | | Explicitly defines default sorting, i.e. ['name', 'desc'] |
```

Comment:
can we add also an example with asc in this way users can understand the string used to change the order